### ISSUE TITLE
Benchmarks

### ISSUE THREAD

**Issue by plwhite** [open-telemetry/opentelemetry.io#3699]
I find the benchmarks page a little hard to follow. It has some graphs of CPU and memory used in various different scenarios, but I couldn't figure out what the different scenarios were. In particular:

- Log10KDPS appears in lots of titles. I'm guessing this means something like "test with 10K logs per second", but it wasn't at all clear
- I couldn't find the definitions of what the different lines are. The linked page talks about various receivers, but the lines on the graph seem to be about different deployment options for what the code is running.
- The units could use being specified; I'm pretty sure it is "percentage an an individual CPU core for this load", for example, but it wasn't clear.

I will rummage through the generation code to figure this out, but I feel a couple of sentences of explanation in the text could make everything far easier for me to follow.

**Comment by svrnm** [comment]
Hey @plwhite , thanks for raising this issue, can you maybe include a link to the Benchmark page you are talking about (I think we have 2 or 3 now).

**Comment by plwhite** [comment]
Sorry, here you go: https://opentelemetry.io/docs/collector/benchmarks/

**Comment by svrnm** [comment]
Thanks for clarification @plwhite 

@open-telemetry/collector-approvers , PTAL!

**Comment by svrnm** [comment]
old issue, but @open-telemetry/collector-approvers PTAL, if this is still relevant

**Comment by mx-psi** [comment]
I think this is still relevant, similar issue: open-telemetry/opentelemetry-collector/issues/14104 cc @codeboten
